adapters/export/otlp/metrics.go

```diff
@@ -0,0 +1,169 @@
+package otlp
+
+import (
+	"context"
+	"fmt"
+	"github.com/kubaceg/sofar_g3_lsw3_logger_reader/adapters/devices/sofar"
+	grpc "go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	http "go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	sdk "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.18.0"
+	"log"
+)
+
+const (
+	appName = "sofar.logger"
```

Comment:
In my opinion this value should be configurable, same as prefix in mqtt configuration